services/streammanager/kafka/kafkamanager.go

```diff
@@ -29,17 +29,18 @@ type avroSchema struct {
 
 // configuration is the config that is required to send data to Kafka
 type configuration struct {
-	Topic         string
-	HostName      string
-	Port          string
-	SslEnabled    bool
-	CACertificate string
-	UseSASL       bool
-	SaslType      string
-	Username      string
-	Password      string
-	ConvertToAvro bool
-	AvroSchemas   []avroSchema
+	Topic             string
+	HostName          string
+	Port              string
+	SslEnabled        bool
+	CACertificate     string
+	UseSASL           bool
+	SaslType          string
+	Username          string
+	Password          string
+	MultiTopicSupport bool
```

Comment:
Why are we introducing this config option if nobody is to use it?

Reply:
Please remove this, it's not being used.

Reply:
This is addressed